lms/djangoapps/instructor/views/tools.py

```diff
@@ -71,7 +72,11 @@ def get_student_from_identifier(unique_student_identifier):
 
     DEPRECATED: use student.models.get_user_by_username_or_email instead.
```

Comment:
What about this message? Is it still valid? Just asking. @andrey-canon

Reply:
I don't know, this uses `student.models.get_user_by_username_or_email`